src/http.zig

```diff
@@ -1408,6 +1408,7 @@ pub const InternalState = struct {
     received_last_chunk: bool = false,
```

Comment:
We probably should have `flags` instead of 5 booleans in a followup/refactor

Reply:
yeah